packages/preferences/src/browser/preferences-tree-widget.ts

```diff
@@ -171,6 +185,12 @@ export class PreferencesContainer extends SplitPanel implements ApplicationShell
         super.onCloseRequest(msg);
         this.dispose();
     }
+
+    public activatePreferenceEditor(preferenceScope: PreferenceScope) {
```

Comment:
`public` is the default, I think it should be internal and called from `onActivateRequest` to be executed on each activation